presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -1484,4 +1493,59 @@ protocol::NodeStatus PrestoServer::fetchNodeStatus() {
   return nodeStatus;
 }
 
+void PrestoServer::convertToVeloxPlan(
```

Comment:
Generally PrestoServer is limited to registration and configuration code. Lets move this function (maybe call it veloxPlanConversion) to another file in presto_cpp/main/types. e.g. The native sidecar function to get functions metadata uses https://github.com/prestodb/presto/blob/master/presto-native-execution/presto_cpp/main/types/FunctionMetadata.h

Reply:
I'm not sure if  `presto_cpp/main/types` is the right place since it doesn't really add anything to the plan conversion and is mostly processing the http request and response. What do you think about moving it to another file in `presto_cpp/main` and calling it VeloxPlanConversion.h/cpp?

Reply:
Since this function is a wrapper over VeloxInteractiveQueryPlanConverter, we can add in the presto_cpp/main/types folder I feel. But we should consider the dependent headers as well. If this code is pulling header files in presto_cpp/main we can keep in there in a separate file.

Reply:
Ok, I think I can separate out the http components to minimize dependencies and keep it clean. It will just need to the memory pool and validator, but that should be fine.

Reply:
done